frontend/src/lib/api/private.ts

```diff
@@ -6,11 +6,62 @@ import { DestructionList } from "./destructionLists";
 import { request } from "./request";
 import { Review } from "./review";
 
-export type ZaaktypeChoice = Option & {
+export type ZaaktypeChoice = Option<string, string> & {
   /** A combination of the identification and the date on which the zaaktype will no longer be valid (if present). */
   extra: string;
 };
 
+/**
+ * Retrieve informatieobjecttypen from Open Zaak and return a value and a label per informatieobjecttype. The label is
+ * the field 'omschrijving'.
+ */
+export async function listInformatieObjectTypeChoices() {
+  return cacheMemo(
+    "listInformatieObjectTypeChoices",
+    async () => {
+      const response = await request("GET", "/_informatieobjecttype-choices/");
+      const promise: Promise<Option[]> = response.json();
+
+      return promise;
+    },
+    [],
+  );
+}
```

Comment:
The informatieobjecttypes need to be only the ones that are linked to the zaaktype selected:
![Screenshot from 2025-01-13 16-47-11](https://github.com/user-attachments/assets/9a1630dd-dc6a-45e3-9404-2264213cce21)
This is also the case for the statustypen and resultaattypen.

The filters to be able to do this are not implemented yet in the backend :S